src/ui/rcb-ui.svelte

```diff
@@ -66,13 +67,17 @@
 
   onMount(() => {
     if (!isInElement) {
-      document.body.style.overflow = "hidden"; // Prevents background scrolling
+      document.documentElement.style.height = "100%";
+      document.body.style.height = "100%";
```

Comment:
Modifying these makes me itchy, feels very invasive towards the hosting application and we risk that we cannot set them back.

Is there no other way to avoid changing the document?

Reply:
Not really. 

The better solution would be to use a MIT dependency that does `position: fixed` saving the scrollTop of the page on iOS or overflow: hidden if not iOS, I have read:

https://github.com/willmcpo/body-scroll-lock

Although this dependency has problems since they shipped a bug in the latest version that doesn't work well, it seems folks are using now https://github.com/tuax/tua-body-scroll-lock for this kind of thing, but I figured we keep if simple. 

At the end, you gonna have to change something on the body, and in the case of `onDestroy()` not being called (hopefully we cover all cases with this PR, at least for our purchase flows, I didn't touch paywalls), you can always refresh the page worst case, but yeah, this is a known pitfall of position fixed modals